### ISSUE TITLE
unknown engine error with mutable matrix and clean

### ISSUE THREAD

**Issue by DanGrayson** [Macaulay2/M2#56]
Reported by:    kroner

Using clean on a MutableMatrix produces unknown engine error.

```
i12 : A = new MutableMatrix from matrix{{1_RR}}

o12 = | 1 |

o12 : MutableMatrix

i13 : clean(0.1,A)
stdio:13:1:(3): error: unknown engine error
```

**Comment by mikestillman** [comment]
This bug still exists, although it works correctly for densely encoded mutable matrices:
```
A = mutableMatrix({{1_RR}}, Dense=>true)
clean(0.1,A) -- works fine
A = mutableMatrix({{1_RR}}, Dense=>false)
clean(0.1,A) -- crashes: not implemented.
```

**Comment by mikestillman** [comment]
I am deferring this to version 1.9.

**Comment by mikestillman** [comment]
Error message is now correctly constructed and returned.  A test was placed into Macaulay2Doc/functions/clean-doc.m2.  Closing this issue.  Fixed in commit ff7473fb874.
